Library/Formula/libnfc.rb

```diff
@@ -31,14 +32,27 @@ def install
 end
 
 __END__
+diff --git a/libnfc/buses/uart.c b/libnfc/buses/uart.c
```

Comment:
This patch can be replaced with: https://github.com/nfc-tools/libnfc/commit/1785ac14c2eab7e66877456845dff29e8b51cb94.diff